spec.avsc

```diff
@@ -1,22 +1,56 @@
 {
-  "namespace":"io.cloudevents",
+  "namespace": "io.cloudevents.avro",
   "type":"record",
   "name":"CloudEvent",
   "version":"1.0",
   "doc":"Avro Event Format for CloudEvents",
   "fields":[
     {
-      "name":"attribute",
-      "type":{
-        "type":"map",
-        "values":[
-          "null",
-          "boolean",
-          "int",
-          "string",
-          "bytes"
-        ]
-      }
```

Comment:
If you take out "attribute", where do you suggest one puts "extensions". As per my understanding, extensions,  mandatory and optional attributes all go under "attribute" as per 1.0 spec version.

Reply:
(sorry for late reply I though this PR was closed and forgotten) 

Indeed I totally forgot extensions when i wrote this a while ago. The Avro specs does not provide a way to do dynamic name when using complex types , A solution could be to actually keep attributes and rename it to "extensions". 

This would actually be clean in the avro's  generated source code  since it would consist to do something like `cloudEvent.getExtensions().get("extensionName")` to get an extension,

Reply:
Can't you add extensions as you have attributes and keep the fields?